### PR TITLE
CLEWS-24031 - doc tracking

### PR DESCRIPTION
**Pull request by seungatGro**
inserting tag manager script as script tag in docs page:

image

(above is on test branch. rebased from development) 
https://developers.gro-intelligence.com/CLEWS-24031-doc-tracking should be up after gh-pages branch builds

### CODE REVIEW

File: docs/_static/js/ga-tag-manager.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ *  google analytics script to add tracking
+ *  sphinx rtd theme uses deprecated version of tag script
+ */
+
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag('js', new Date());
+
+gtag('config', 'UA-49099862-1');
```
Comment (muzigao) [inline docs/_static/js/ga-tag-manager.js]: do they support using google tag manager?

Comment (muzigao) [reply docs/_static/js/ga-tag-manager.js]: I mean should we refer to google tag container id instead of GA id?

Comment (seungatGro) [reply docs/_static/js/ga-tag-manager.js]: i was under the impression that tag manager only captures website data. updated. removed custom js, just extended the sphinx rtd theme layout so we input extra <head> and <body> elements

Comment (seungatGro) [reply docs/_static/js/ga-tag-manager.js]: image

File: docs/conf.py [hunk 2]
```diff
@@ -90,7 +90,7 @@
 
 # still build other branches, but hide the version selectors in _static/css/custom-theme.css.
 # Uncomment this line to stop building them altogether:
-# scv_whitelist_branches = ('development',)
+scv_whitelist_branches = ('CLEWS-23982', 'CLEWS-24031', 'development',)
```
Comment (seungatGro) [inline docs/conf.py]: revert before merge to dev branch only

**Review by muzigao — APPROVED** [review]

**Pull request merged**